src/components/QuizApp.js

```diff
@@ -32,48 +32,48 @@ class QuizApp extends Component {
   }
   componentDidMount(){
         this.loadQuizData();
-      }
-
-      componentDidUpdate(prevProps, prevState) {
-        if (this.state.currentQuestion !== prevState.currentQuestion) {
-          this.setState(() => {
-            return {
-              questions: quizData[this.state.currentQuestion].question,
-              options: quizData[this.state.currentQuestion].options,
-              answer: quizData[this.state.currentQuestion].answer
-            };
-          });
-        }
-      }
+  }
 
-    handleCheckAnswer=(myAnswer)=>{
-      this.setState({
-        myAnswer: myAnswer
-      })
+  componentDidUpdate(prevProps, prevState) {
+    if (this.state.currentQuestion !== prevState.currentQuestion) {
+      this.setState(() => {
+        return {
+          questions: quizData[this.state.currentQuestion].question,
+          options: quizData[this.state.currentQuestion].options,
+          answer: quizData[this.state.currentQuestion].answer
+        };
```

Comment:
Można użyć destrukturyzacji {quiz, options, answer} = quizData[this.state.currentQuestion], a następnie zwrócić {quiz, options, answer}